bazarr.py

```diff
@@ -150,23 +128,30 @@ def daemon(bazarr_runner=lambda: start_bazarr()):
             else:
                 bazarr_runner()
 
+    def should_stop():
+        return daemonStatus.should_stop()
 
-    bazarr_runner = lambda: start_bazarr()
-
-    should_stop = lambda: False
+    def bazarr_runner():
+        start_bazarr(daemonStatus)
 
     daemonStatus = DaemonStatus()
 
+    def force_shutdown():
+        # force the killing of children processes
+        daemonStatus.force_stop()
+        # if a new SIGTERM signal is caught the standard behaviour should be followed
+        signal.signal(signal.SIGTERM, signal.SIG_DFL)
+        # emulate a Ctrl C command on itself (bypasses the signal thing but, then, emulates the "Ctrl+C break")
+        os.kill(os.getpid(), signal.SIGINT)
+
     def shutdown():
         # indicates that everything should stop
         daemonStatus.stop()
-        # emulate a Ctrl C command on itself (bypasses the signal thing but, then, emulates the "Ctrl+C break")
-        os.kill(os.getpid(), signal.SIGINT)
+        # if a new sigterm signal is caught it should force the shutdown of children processes
+        signal.signal(signal.SIGTERM, lambda signal_no, frame: force_shutdown())
```

Comment:
We update the SIGTERM signal handling policy to force the shutdown so if the SIGTERM signal is sent once more it will actually send a SIGTERM signal to children processes.